internal/server/server.go

```diff
@@ -46,13 +46,60 @@ type Server struct {
 	logger          log.Logger
 	instrumentation *telemetry.Instrumentation
 	sseManager      *sseManager
+	ResourceManager *ResourceManager
```

Comment:
```suggestion
	resourceManager *ResourceManager
```

By default it's always good to not make it exported. Can update it when we need it to be exported :)